time stamp

# base (shared) values
int32 test_number
bool default_control
string flight_mode
bool test_finished
bool coord_ok

# StatusSecondary
string control_mode  # did something break?
bool regulate_finished
string description  # for telemetry description
